### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by micedreams**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Remove lint errors

## Featured Covered in this PR
- [X] [Remove unused variables.](https://github.com/SankethBK/diaryvault/commit/5354bd600054428a3d78ab3c195e1e64f581091c)
- [X] [Remove override.](https://github.com/SankethBK/diaryvault/commit/26c3612f0764a97b0f944c3d96e5475c9a273ab4)
- [X] [Return explicit null from methods that do nothing.](https://github.com/SankethBK/diaryvault/commit/d3d76f816773b0cebc9f2738ac6abd398a0c9deb)
- [X] [Fix null check.](https://github.com/SankethBK/diaryvault/commit/ab0d69592c3640053d696b790651b4cd3c6dde5b)
- [X] [Make all the constructor parameters final.](https://github.com/SankethBK/diaryvault/commit/62dad15f0c6fbae817ba3819e96786d1fa57567d)

### CODE REVIEW

**Comment by SankethBK** [comment]
Let me know if this is ready, I'll review it

**Comment by micedreams** [comment]
> Let me know if this is ready, I'll review it

It is!!

File: lib/features/notes/presentation/widgets/note_preview_card.dart [hunk 1]
```diff
@@ -22,6 +21,8 @@ class NotePreviewCard extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    late bool isSelected;
```
Comment (SankethBK) [inline lib/features/notes/presentation/widgets/note_preview_card.dart:24]: late is not necessary here as `isSelected` is local variable

**Review by SankethBK — APPROVED** [review]
Minor comment, rest all looks good

**Pull request merged**